frontend/src/router.ts

```diff
@@ -305,4 +305,29 @@ export const routerBeforeEach: NavigationGuard = async (to, from, next) => {
   next();
 };
 
-router.beforeEach(routerBeforeEach);
+export const redirectSections: NavigationGuard = async (to, from, next) => {
+  if (_.get(to, "meta.activator", false)) {
```

Comment:
For the sake of readability I'd negate this condition and `return next()` here to avoid long `if` statement.

Reply:
Also, `false` default value seems redundant, and moreover it confuses, looks like we're comparing `meta.activator` to `false`. I guess getting `undefined` would work as well.